index.html

```diff
@@ -3,19 +3,60 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
+    <title> <img src="favicon.ico" alt="Karma logo">Karma</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
+</head> 
+
+<header>
+</header>
+
+
+<nav>
+        <img src="favicon.ico" alt="Karma logo">
+        <ul class="navbut">
+        
+            <li>Meet Karma</li>
+            <li>How it Works </li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+
```

Comment:
Nice use of an unordered list. Can you add links inside each item, and give them a hover effect?